server.rb

```diff
@@ -41,35 +37,26 @@ def language(key)
       sender_id = messaging["sender"]["id"]
       text = messaging["message"]["text"]
 
-      # Logic for if to send the initial message to user
-      # Need to work on adding this feature to EACH INDIVIDUAL USER
-        unless @@users[sender_id]
-          Bot.new.send_message(sender_id, "To what language would you like me to translate? I know English, Vietnamese, & French")
-        end
+      unless @@users[sender_id]
+        Bot.new.send_message(sender_id, "To what language would you like me to translate? I know a lot")
+        Bot.new.send_message(sender_id, "Please respond with '/language'. For example, '/English'")
+      end
 
-        @@users[sender_id] ||= []
+      @@users[sender_id] ||= "English"
 
-        # Language choice
-        if text == "/Vietnamese" || (text == "Vietnamese" && @@initial <= 1)
-          Bot.new.send_message(sender_id, "Ok, I'll translate to Vietnamese. If you want to change later simply say '/Language'")
-          @@users[sender_id] = "Vietnamese"
-          @@initial += 1
-        elsif text == "/English" || (text == "English" && @@initial <= 1)
-          Bot.new.send_message(sender_id, "Ok, I'll translate to English. If you want to change later simply say '/Language'")
-          @@users[sender_id] = "English"
-          @@initial += 1
-        elsif text == "/French" || (text == "French" && @@initial <= 1)
-          Bot.new.send_message(sender_id, "Ok, I'll translate to French. If you want to change later simply say '/Language'")
-          @@users[sender_id] = "French"
-          @@initial += 1
-        else
-        end
+      if text[0] == "/"
+        @@users[sender_id] = text[1..-1]
+      end
 
-      reply = "You said: #{text}"
+      choice = language(sender_id)
 
-      # Bot responds with translation language of users choice.
-      reply = EasyTranslate.translate(text, language(sender_id))
-      Bot.new.send_message(sender_id, reply)
+      if choice == nil
+        Bot.new.send_message(sender_id, "Sorry couldn't find #{@@users[sender_id]} so I'm going to speak English for now.")
+        Bot.new.send_message(sender_id, "Please try another language with '/language'. For example, '/English' for translations to English")
+      else
+        reply = EasyTranslate.translate(text, choice)
```

Comment:
@PrimeTimeTran why don't you use `EasyTranslate.translate(text, to: choice)` and make the `language(sender_id)` method returns just a language.